services/user.py

```diff
@@ -0,0 +1,48 @@
+from db.models import User

```

Comment:
You're importing the User model directly, which is not a best practice. You should use `get_user_model()` from `django.contrib.auth` to get the user model.